package.json

```diff
@@ -14,6 +14,7 @@
     "typecheck": "graphql-codegen && tsc --noEmit --pretty",
     "lint:check": "eslint . --max-warnings=1500",
     "lint:fix": "eslint . --fix",
+    "lint-staged": "lint-staged",
```

Comment:
`lint-staged` is added as a dependency twice. Please remove it once.

Reply:
This is a script that is used to run npm run lint-staged as recommended by you, not a dependency.